go/vt/vtgate/planbuilder/grouping.go

```diff
@@ -23,8 +23,8 @@ import (
 	"vitess.io/vitess/go/vt/vtgate/engine"
 )
 
-func planGroupBy(pb *primitiveBuilder, input logicalPlan, groupBy sqlparser.GroupBy) (logicalPlan, error) {
-	if len(groupBy) == 0 {
+func planGroupBy(pb *primitiveBuilder, input logicalPlan, groupBy *sqlparser.GroupBy) (logicalPlan, error) {
```

Comment:
I think we can just send in []Expr here instead of groupBy

Reply:
But then the group by being nil will have to checked outside where the function is called from. Is there a reason why we want to send []Expr instead of a GroupBy